### PR TITLE
DB-6235: Update starter kit READMEs

### PR DESCRIPTION
**Pull request by CobyPear**
## What changes were made?
- Create partials for shared readme sections
- Compose starter readmes with partials
## Where were the changes made?
CLI templates

## How have the changes been tested?
locally
## Additional information

Don't forget to [add a changeset](https://github.com/pantheon-systems/decoupled-kit-js#generating-a-changeset) if needed!

### CODE REVIEW

File: packages/create-pantheon-decoupled-kit/src/templates/partials/shared/readme/sharedReadme.ts [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+/**
+ * Explains the reason for using GET requests in GraphQL and how to change to POST
+ */
+export const graphqlPostRequest =
+	() => /* md */ `### Use \`POST\` for GraphQL requests
+
+This starter uses \`GET\` for GraphQL requests by default in order to utilize the [WPGraphQL Smart Cache Network Cache](https://github.com/wp-graphql/wp-graphql-smart-cache/blob/main/docs/network-cache.md#network-cache). Editing this
+configuration to use \`POST\` requests can be done in \`/lib/WordPressClient.js\`.
+
+To achieve this, set each \`GraphQLClientFactory\` constructors \`method\` parameter
```
Comment (backlineint) [inline packages/create-pantheon-decoupled-kit/src/templates/partials/shared/readme/sharedReadme.ts:165]: I think this should be possessive: `constructor's`

File: packages/create-pantheon-decoupled-kit/src/templates/partials/shared/readme/sharedReadme.ts [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+ * The Getting Started section for the starter kit READMEs
+ * @param starterName one of Next WordPress, Next Drupal, or Gatsby WordPress
+ */
+export const gettingStarted = (
+	starterName: 'Next WordPress' | 'Next Drupal' | 'Gatsby WordPress',
+) => /* md */ `## Getting Started
+
+The ${starterName} starter requires Node.js and is built and tested on the LTS version which can be found on the [Node.js downloads page](https://nodejs.org/en/download).
+
+The starter kit supports npm, pnpm, and yarn. If you created the starer via the Pantheon Front-End Sites dashboard, it will use npm by default. To change this, delete the \`package-lock.json\` \`node-modules\` folder, then run the install command of your preferred package manager.
```
Comment (backlineint) [inline packages/create-pantheon-decoupled-kit/src/templates/partials/shared/readme/sharedReadme.ts:31]: This reads a little funny as rendered: delete the `package-lock.json node-modules` folder. Maybe something like:

delete the `package-lock.json` file and the `node-modules` folder

File: packages/create-pantheon-decoupled-kit/src/templates/partials/shared/readme/sharedReadme.ts [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+type CmsType = 'drupal' | 'wordpress';
+
+/**
+ * The header section for the starter kit READMEs
+ */
+export const readmeHeader =
+	() => /* md */ `<div style="display:flex;flex-direction:column">
+	<img src="https://raw.githubusercontent.com/pantheon-systems/decoupled-kit-js/canary/web/static/img/B_Fist-Tagline.png" height="120" style="background:#ffffff;border-radius:8px;margin:auto;display:flex;" alt="Pantheon.io logo featuring a fist capturing lighting. Pantheon™, The Platform for Extraordinary Websites.">
+	<a href="https://pantheon.io">
+		<h1 style="margin:auto;" align="center">Pantheon</h1>
```
Comment (backlineint) [inline packages/create-pantheon-decoupled-kit/src/templates/partials/shared/readme/sharedReadme.ts:13]: I don't think the h1 here should be Pantheon. I think it should instead be the name of the starter kit. Moving that outside of this component might make it easier to vary this heading.

**Review by backlineint — CHANGES_REQUESTED** [review]
This looks like a nice improvement to me. Just a few relatively small changes mentioned inline.

File: packages/create-pantheon-decoupled-kit/src/templates/partials/shared/readme/sharedReadme.ts (hunk 3 above)
Comment (CobyPear) [reply packages/create-pantheon-decoupled-kit/src/templates/partials/shared/readme/sharedReadme.ts]: Makes sense to me!

**Review by backlineint — APPROVED** [review]

**Comment by backlineint** [comment]
Test status isn't coming back, likely due to https://status.circleci.com/incidents/14q5x3wmz71m

Will check in on this later.

**Pull request merged**